src/renderer/components/archive/entry-view.js

```diff
@@ -29,14 +29,19 @@ const EntryView = ({ entry, intl }) => (
     </h6>
     {entry.meta.length > 0 ? (
       <div className={metaWrapper}>
-        {entry.meta.map(meta => (
-          <div className={formRow} key={meta.key}>
-            <div className={labelWrapper}>{meta.key}</div>
-            <div className={inputWrapper}>
-              <Copyable>{meta.value}</Copyable>
-            </div>
-          </div>
-        ))}
+        {entry.meta.map(
+          meta =>
+            Object.keys(meta).length ? (
```

Comment:
You could use jsx control statements here:

```javascript
<If condition={Object.keys(meta).length > 0}>
  <!-- content -->
</If>
```

Reply:
This might be more tidy than the ternary operator.

Reply:
Oh cool, didn't know that :)

Reply:
Yeah it's great! Much nicer imo to read just XML..